packages/mco/components/discovered-application-wizard/utils/k8s-utils.ts

```diff
@@ -135,6 +143,11 @@ export const createPromise = (
     })
   );
 
+  let labels: ObjectMetadata['labels'];
+  const annotations = isVolumeConsistencyEnabled && {
+    [IS_CG_ENABLED_ANNOTATION]: 'true',
+  };
```

Comment:
```suggestion
  } : {};
```
I would prefer ternary operator to make things clear. You are applying an annotation only when it is enabled else annotations is empty. Also are we sure there will be no other annotations?